src/taipy/templates/scenario-management/{{cookiecutter.__root_folder_name}}/pages/root.md

```diff
@@ -9,8 +9,13 @@
 |>
 |>
 
-<|part|class_name=main|render={selected_scenario}|
+<|part|class_name=main|
```

Comment:
What is the use of class_name=main

Reply:
IDK. Is'nt it used for applying default styling?

Reply:
I don't think so